ReactiveCocoa.podspec

```diff
@@ -60,7 +60,10 @@ Pod::Spec.new do |s|
     ]
     ss.osx.framework = 'AppKit'
 
-    ss.tvos.source_files = '**/ReactiveCocoa.h'
+    ss.tvos.source_files = [
+      '**/ReactiveCocoa.h',
+      'ReactiveCocoa/**/*{UIButton,UICollectionReusableView,UIControl,UIGestureRecognizer,UISegmentedControl,UITableViewCell,UITableViewHeaderFooterView,UIText}*'
```

Comment:
Opened https://github.com/mhuusko5/Podspecs/issues/7 and will try to get to it today.